scripts/add_ts_data_bidmatrixinfo.py

```diff
@@ -0,0 +1,59 @@
+###this script adds time series data to the bid matrix information objects in the powerops client
+
+from cognite.client.data_classes import TimeSeries
+from cognite.powerops import PowerOpsClient
+import random
+import datetime
+
+
+power = PowerOpsClient.from_settings()
+cdf = power.cdf
+
+start_time = int((datetime.datetime.now() - datetime.timedelta(days=365)).timestamp() * 1000)
+end_time = int(datetime.datetime.now().timestamp() * 1000)
+
+# creating time series, with data points every hour
+if not cdf.time_series.retrieve(external_id="wim_ts_hourly"):
+    wim_ts_hourly = cdf.time_series.create(TimeSeries(name="wim_ts_hourly", external_id="wim_ts_hourly"))
+    data_points = [
+        {"timestamp": start_time + i * 3600 * 1000, "value": random.random()}
+        for i in range(int((end_time - start_time) / (3600 * 1000)))
+    ]
+    cdf.time_series.data.insert(datapoints=data_points, external_id="wim_ts_hourly")
+
+# creating an empty time series, without a name to check the external_id default
+if not cdf.time_series.retrieve(external_id="wim_ts_empty"):
+    wim_ts_empty = cdf.time_series.create(TimeSeries(external_id="wim_ts_empty"))
+
+tsList = ["wim_ts_hourly", "wim_ts_empty"]
+
+
+def get_random_timeseries():
+    choice = random.choice([1, 2, 3])
+    if choice == 1:
+        return [random.choice(tsList)]  # Velg en tidsserie
+    elif choice == 2:
+        return random.sample(tsList, 2)  # Velg to tidsserier
+    else:
+        return tsList
```

Comment:
**thought:** this function isn't very expandable currently, what if I decide to add a 3rd or 4th time series to the `tsList`?